test/models/user_test.rb

```diff
@@ -0,0 +1,43 @@
+require 'test_helper'
+
+class UserTest < ActiveSupport::TestCase
+  def setup
+    @user = User.new(name: "Example User", email: "user@example.com", password: "foobar", password_confirmation: "foobar")
```

Comment:
this line is starting to get too long which negatively affects readability.

I'd suggest splitting it into multiple lines, ex: 
```rb
    @user = User.new(
      name: "Example User",
      ...
    )
```

You can using a formatter to automate this in your editor. We use [Rubocop](https://github.com/rubocop/rubocop) in the main Docfox repo and there are others like Rufo.

Something I also find useful is to add a visual indicator at the 80-character line which helps me decide when to start breaking up a long line into multiple lines. For instance in VS Code you can get this by adding "editor.rulers": [80] to the settings (Cmd+,) but other editors will have a similar feature